testament/important_packages.nim

```diff
@@ -10,150 +10,144 @@ packages listed here should ideally have regularly updated release tags, so that
-pkg1 "gnuplot", "nim c gnuplot.nim"
-# pkg1 "gram", "nim c -r --gc:arc --define:danger tests/test.nim", "https://github.com/disruptek/gram"
+pkg "cligen", "nim c --path:. -r cligen.nim"
+pkg "combparser", "nimble test --gc:orc"
+pkg "compactdict"
+pkg "comprehension", "nimble test", "https://github.com/alehander42/comprehension"
+# pkg "criterion" # pending https://github.com/disruptek/criterion/issues/3 (wrongly closed)
+pkg "dashing", "nim c tests/functional.nim"
+pkg "delaunay"
+pkg "docopt"
+pkg "easygl", "nim c -o:egl -r src/easygl.nim", "https://github.com/jackmott/easygl"
+pkg "elvis"
+# pkg "fidget" # pending https://github.com/treeform/fidget/issues/133
+pkg "fragments", "nim c -r fragments/dsl.nim"
+pkg "fusion"
+pkg "gara"
+pkg "glob"
+pkg "ggplotnim", "nim c -d:noCairo -r tests/tests.nim"
+# pkg "gittyup", "nimble test", "https://github.com/disruptek/gittyup"
+pkg "gnuplot", "nim c gnuplot.nim"
+# pkg "gram", "nim c -r --gc:arc --define:danger tests/test.nim", "https://github.com/disruptek/gram"
   # pending https://github.com/nim-lang/Nim/issues/16509
-pkg1 "hts", "nim c -o:htss src/hts.nim"
-# pkg1 "httpauth"
-pkg1 "illwill", "nimble examples"
-pkg1 "inim"
-pkg1 "itertools", "nim doc src/itertools.nim"
-pkg1 "iterutils"
-pkg1 "jstin"
-pkg1 "karax", "nim c -r tests/tester.nim"
-pkg1 "kdtree", "nimble test", "https://github.com/jblindsay/kdtree"
-pkg1 "loopfusion"
-pkg1 "macroutils"
-pkg1 "manu"
-pkg1 "markdown"
-pkg1 "memo"
-pkg1 "msgpack4nim", "nim c -r tests/test_spec.nim"
+pkg "hts", "nim c -o:htss src/hts.nim"
+# pkg "httpauth"
+pkg "illwill", "nimble examples"
+pkg "inim"
+pkg "itertools", "nim doc src/itertools.nim"
+pkg "iterutils"
+pkg "jstin"
+pkg "karax", "nim c -r tests/tester.nim"
+pkg "kdtree", "nimble test", "https://github.com/jblindsay/kdtree"
+pkg "loopfusion"
+pkg "macroutils"
+pkg "manu"
+pkg "markdown"
+pkg "memo"
+pkg "msgpack4nim", "nim c -r tests/test_spec.nim"
 
 # these two are special snowflakes
-pkg1 "nimcrypto", "nim c -r tests/testall.nim"
-pkg1 "stint", "nim c -o:stintt -r stint.nim"
+pkg "nimcrypto", "nim c -r tests/testall.nim"
+pkg "stint", "nim c -o:stintt -r stint.nim"
 
-
-# packages N-Z
-pkg2 "nake", "nim c nakefile.nim"
-pkg2 "neo", "nim c -d:blas=openblas tests/all.nim"
-# pkg2 "nesm", "nimble tests" # notice plural 'tests'
-# pkg2 "nico"
-pkg2 "nicy", "nim c -r src/nicy.nim"
-pkg2 "nigui", "nim c -o:niguii -r src/nigui.nim"
-pkg2 "NimData", "nim c -o:nimdataa src/nimdata.nim"
-pkg2 "nimes", "nim c src/nimes.nim"
-pkg2 "nimfp", "nim c -o:nfp -r src/fp.nim"
+pkg "nake", "nim c nakefile.nim"
+pkg "neo", "nim c -d:blas=openblas tests/all.nim"
+# pkg "nesm", "nimble tests" # notice plural 'tests'
+# pkg "nico"
+pkg "nicy", "nim c -r src/nicy.nim"
+pkg "nigui", "nim c -o:niguii -r src/nigui.nim"
+pkg "NimData", "nim c -o:nimdataa src/nimdata.nim"
+pkg "nimes", "nim c src/nimes.nim"
+pkg "nimfp", "nim c -o:nfp -r src/fp.nim"
 when false:
```

Comment:
IMHO `when false:` should not be allowed, instead remove the line,Git has history and revert if needed.

Reply:
it's pre-existing, and simply removing it is worse; readers of this file should be able to tell at a glance what was disabled and why (with a link to corresponding issue which may live in a different repo)

Reply:
pushed another comment which uses a commented out line, as other disabled packages do